src/main/pom.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>groupId</groupId>
+    <artifactId>main</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    
+</project>
```

Comment:
is this change really necessary, normally there are no poms at this level? if not could you force push without it pls?

Reply:
Apologies. I'll revert the pull-request and send another one with only code change without pom. I had done that in my fork to differentiate from your releases. But I get your point.